data/templates/snom.tmpl

```diff
@@ -379,7 +379,7 @@
         <subscription_delay perm="">0</subscription_delay>
         <terminate_subscribers_on_reboot perm="">on</terminate_subscribers_on_reboot>
         <show_ivr_digits perm="">off</show_ivr_digits>
-        <settings_refresh_timer perm="">3600</settings_refresh_timer>
+        <settings_refresh_timer perm="">{% if provisioning_complete %}0{% else %}60{% endif %}</settings_refresh_timer>
```

Comment:
Expressions are more compact than flow control statements. The latter suits better multi line code blocks.

```suggestion
        <settings_refresh_timer perm="">{{ provisioning_complete ? '0' : '60' }}</settings_refresh_timer>
```